k9mail/src/main/java/com/fsck/k9/activity/MessageLoaderHelper.java

```diff
@@ -416,7 +421,7 @@ public void loadMessageRemoteFailed(Account account, String folder, String uid,
         void onMessageDataLoadFailed();
 
         void onMessageViewInfoLoadFinished(LocalMessage localMessage, MessageViewInfo messageViewInfo);
-        void onMessageViewInfoLoadFailed(LocalMessage localMessage);
+        void onMessageViewInfoLoadFailed(LocalMessage localMessage, MessageViewInfo messageViewInfo);
```

Comment:
The goal is to get rid of having to pass (Local)Message to the view. So being able to remove it here is a win.
Also, there's no need to be consistent with the arguments. We're not writing an API for public consumption. If it's not used, we should remove it.